operating_unit/models/res_users.py

```diff
@@ -29,51 +53,32 @@ def operating_unit_default_get(self, uid2=False):
 
     @api.model
     def _default_operating_unit(self):
-        return self.operating_unit_default_get()
+        return self._get_default_operating_unit()
 
     @api.model
     def _default_operating_units(self):
         return self._default_operating_unit()
 
```

Comment:
Is it possible to use a single method instead ?

Reply:
I think so. Code updated.
Thanks for the review 👍